packages/@aws-cdk/aws-appmesh/test/test.route.ts

```diff
@@ -62,6 +73,10 @@ export = {
           match: {
             serviceName: 'test.svc.local',
           },
+          timeout: {
+            idle: cdk.Duration.seconds(10),
+            perRequest: cdk.Duration.seconds(10),
+          },
```

Comment:
You need to actually assert the configuration in the `expect(...` steps.

Reply:
sweet ack